parachain/runtime/src/lib.rs

```diff
@@ -106,10 +106,10 @@ pub mod opaque {
 
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("artemis-node"),
-	impl_name: create_runtime_str!("artemis-node"),
+	spec_name: create_runtime_str!("snowbridge"),
```

Comment:
will need to rename stuff in the cumulus node to be called snowbridge too

Reply:
You mean the name of the binary? If so, sure yeah that will happen in time.

For more context, The polkadot webapp specifically uses `spec_name` to apply our custom types when someone browses our chain: See https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/api/spec/index.ts#L59 for example.